UnleashClient/__init__.py

```diff
@@ -412,7 +412,8 @@ def destroy(self) -> None:
                 engine=self.engine,
             )
 
-        self.unleash_scheduler.shutdown()
+        if getattr(self.unleash_scheduler, "state", None) == STATE_RUNNING:
```

Comment:
This feels wrong. I want to say the right way to do this is to try and shut down the scheduler and if you're worried then trap the SchedulerNotRunningError exeption

Reply:
Maybe broad 'catch' would be better?